### PR TITLE
Set datepicker on birthdate in user profile

### PR DESCRIPTION
**Pull request by xillibit**
Pull Request for Issue # .

### CODE REVIEW

**Comment by betteryouthanme** [comment]
input-append is out of use in bs3. Should be "input-group".
Also the input's class is missing "form-control" and "span12" can be removed.

In the kunena.css is a margin-left defined which spoils the layout for div id birthdate. (Line 882)

Datepicker starts with year 1. (both crypsisb3 and b2) See below:
birthdate-datepicker

**Comment by xillibit** [comment]
I have updated it you can check a new time if you want

File: src/components/com_kunena/template/crypsisb3/assets/js/profile.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ * @package Kunena.Template.Crypsis
+ *
+ * @copyright (C) 2008 - 2016 Kunena Team. All rights reserved.
+ * @license http://www.gnu.org/copyleft/gpl.html GNU/GPL
+ * @link https://www.kunena.org
+ **/
+
+jQuery(document).ready(function() {
+  if (jQuery.fn.datepicker != undefined) {
+		jQuery('#birthdate .input-append.date').datepicker({
```
Comment (betteryouthanme) [inline src/components/com_kunena/template/crypsisb3/assets/js/profile.js]: input-group instead input-append

File: src/components/com_kunena/template/crypsisb3/layouts/user/edit/profile/default.php [hunk 2]
```diff
@@ -33,17 +37,12 @@
...
-					<input id="birthdate" class="form-control" type="text" size="4" maxlength="4"  name="birthdate1"
-					       value="<?php echo $this->escape($birthYear); ?>" />
-					<input class="form-control" type="text" size="2" maxlength="2" name="birthdate2"
-					       value="<?php echo $this->escape($birthMonth); ?>" />
-					<input class="form-control" type="text" size="2" maxlength="2" name="birthdate3"
-					       value="<?php echo $this->escape($birthDay); ?>" />
-				</span>
+				<div id="birthdate">
+        <div class="input-group date">
+          <input type="text" name="birthdate" data-date-format="mm/dd/yyyy" value="<?php echo $this->profile->birthdate == '0001-01-01' ? JFactory::getDate()->format('m/d/Y') : $this->profile->birthdate; ?>"> <span class="input-group-addon"><i class="glyphicon glyphicon-th"></i></span>
```
Comment (betteryouthanme) [inline src/components/com_kunena/template/crypsisb3/layouts/user/edit/profile/default.php]: class="input-group" is missing

Comment (xillibit) [inline src/components/com_kunena/template/crypsisb3/layouts/user/edit/profile/default.php]: The class="input-group" is already on the div

Comment (betteryouthanme) [inline src/components/com_kunena/template/crypsisb3/layouts/user/edit/profile/default.php]: Sorry meant "form-control" on input

**Comment by xillibit** [comment]
@betteryouthanme : i have updated it, it should be ok now

**Comment by betteryouthanme** [comment]
@xillibit yes all fine now

**Pull request merged**